Scraps/Services/RaffleService.cs

```diff
@@ -192,9 +192,14 @@ public async Task StartAsync()
 
                 Broadcast("Waiting to scan again");
 
-                _log.Info("All raffles have been entered, scanning again after {Delay} seconds", _scanDelay / 1000);
+                System.Random _randomifier = new System.Random();
```

Comment:
A single System.Random instance should be a private field on the service instance.